docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.color-pairings.scss

```diff
@@ -3,18 +3,30 @@
 // .example {
 //   @include hb-pairing-color('background-color', 'primary');
 // }
-@mixin hb-pairing-color($property, $color-swatch) {
+@mixin hb-pairing-color($property, $color-swatch, $important: false) {
   @include hb-colorful {
-    // Fallback for browsers that do not support CSS variables
-    #{$property}: hb-get-pairing-color($color-swatch, $hb-colorful-default, $hc-colorful-pairings);
+    @if ($important) {
+      // Fallback for browsers that do not support CSS variables
```

Comment:
do we need // scss-lint:disable ImportantRule here ?

Reply:
🤷‍♀ Probably